girder/molecules/setup.py

```diff
@@ -7,6 +7,7 @@
     packages=find_packages(),
     install_requires=[
       'girder>=3.0.0a5',
+      'girder-jobs',
```

Comment:
Do we need to pip install this in the docker container?

Reply:
This works, but it installs girder-jobs from the web rather than the one in /girder/plugins. We should probably install the one in /girder/plugins to make the jobs plugin version matches the girder version.

Reply:
of course, that is fine.